### PR TITLE
Add foreman_webhooks 0.0.1 - DEB

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* Nightly
* 2.3

### CODE REVIEW

**Comment by lzap** [comment]
It's failing with `LoadError: Error loading the 'postgresql' Active Record adapter. Missing a gem it depends on? pg is not part of the bundle. Add it to your Gemfile.` and I can't figure out why. Honestly, I took discovery gem and just replaced the name. It should use the null-db driver.... hmm.

File: plugins/ruby-foreman-webhooks/debian/changelog [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ruby-foreman-discovery (0.0.1) unstable; urgency=low
```
Comment (evgeni) [inline plugins/ruby-foreman-webhooks/debian/changelog:1]:
```suggestion
ruby-foreman-webhooks (0.0.1) unstable; urgency=low
```

dpkg fails to properly unpack that source package, and thus you get wild errors nobody can understand :)

**Review by evgeni — CHANGES_REQUESTED** [review]
reviewed with @baboon92, you have confused dpkg :)

File: plugins/ruby-foreman-webhooks/debian/copyright [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ruby-foreman-webhooks
+Source: https://github.com/theforeman/foreman_webhooks
+
+Files: *
+Copyright: 2012-2013 Red Hat Inc. <foreman-dev@googlegroups.com>
+License: GPL-3.0+
+
+Files: debian/*
+Copyright: 2013 Red Hat Inc. <foreman-dev@googlegroups.com>
```
Comment (ekohl) [inline plugins/ruby-foreman-webhooks/debian/copyright:10]: That's a while back.

**Comment by lzap** [comment]
Shoot me in my head, but we detect plugin name from a CHANGELOG :-) Wow, well. Okay! Love it.

**Comment by lzap** [comment]
The package looks good on my system.

File: plugins/ruby-foreman-webhooks/debian/compat [hunk 3]
```diff
@@ -0,0 +1 @@
+8
```
Comment (evgeni) [inline plugins/ruby-foreman-webhooks/debian/compat:1]:
```suggestion
11
```

File: plugins/ruby-foreman-webhooks/debian/control [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Source: ruby-foreman-webhooks
+Section: ruby
+Priority: extra
+Maintainer: Lukas Zapletal <lzap+rpm@redhat.com>
+Build-Depends: debhelper (>= 8.0.0), foreman (>= 1.23.0~rc1), foreman-assets(>= 1.23.0~rc1), foreman-nulldb (>= 1.23.0~rc1)
```
Comment (evgeni) [inline plugins/ruby-foreman-webhooks/debian/control:5]:
```suggestion
Build-Depends: debhelper (>= 11~), foreman (>= 1.23.0~rc1), foreman-assets(>= 1.23.0~rc1), foreman-nulldb (>= 1.23.0~rc1)
```

the RPM says it needs Foreman 2.3, here you say 1.23, what's correct? :)

File: plugins/ruby-foreman-webhooks/debian/control [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+Standards-Version: 3.9.3
+Homepage: https://github.com/theforeman/foreman_webhooks
+
+Package: ruby-foreman-webhooks
+Architecture: all
+Depends: ${misc:Depends}, bundler, foreman (>= 1.23.0~rc1)
```
Comment (evgeni) [inline plugins/ruby-foreman-webhooks/debian/control:11]: same, should this be 2.3?

**Review by evgeni — CHANGES_REQUESTED** [review]
minor nitpicks, ack when adressed

File: plugins/ruby-foreman-webhooks/debian/control (hunk 5 above)
Comment (lzap) [reply plugins/ruby-foreman-webhooks/debian/control:11]: Oh yes.

**Comment by lzap** [comment]
Updated the version to 2.3, squashed your changes, thanks.

**Review by evgeni — APPROVED** [review]

**Pull request merged**